sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -815,11 +815,22 @@ async def delete_all_items_by_partition_key(
         partition_key: Union[str, int, float, bool],
         **kwargs: Any
     ) -> None:
-        """Exposes an API to delete all items with a single partition key without the user having
-                 to explicitly call delete on each record in the partition key.
+        """The delete by partition key feature is an asynchronous, background operation that allows you to delete all
+        documents with the same logical partition key value, using the Cosmos SDK. The delete by partition key
+        operation is constrained to consume at most 10% of the total
+        available RU/s on the container each second. This helps in limiting the resources used by
+        this background task.
```

Comment:
May be we can also link to our public docs for this and patch API as well if not already.

Reply:
The links for both are in the Changelog with their respective entries.